src/client/Microsoft.Identity.Client/TelemetryCore/OpenTelemetry/OpenTelemetryClient.cs

```diff
@@ -1,20 +1,20 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-//#if NET6_0_OR_GREATER || NET461_OR_GREATER || NET20_OR_GREATER
 using System;
+using System.Collections.Generic;
 using System.Diagnostics;
-using System.Diagnostics.Metrics;
 using Microsoft.Identity.Client.Core;
+#if NETSTANDARD || NET6_0 || NET462
+using System.Diagnostics.Metrics;
+#endif
 using Microsoft.Identity.Client.Internal;
-using Microsoft.Identity.Client.Internal.Requests;
-//#endif
 
 namespace Microsoft.Identity.Client.TelemetryCore.OpenTelemetry
 {
     /// <summary>
     /// Class to hold the OpenTelemetry objects used by MSAL.
     /// </summary>
-    public class OpenTelemetryClient
+    internal class OpenTelemetryClient : IOpenTelemetryClient
```

Comment:
A more interesting pattern is to have the a `NullOtelClient` and this one, both implementing `IOpenTelemetryClient`. 

Then PlatformProxy would either create a `NullOtelClient` or a `OpenTelemetryClient` depending on the target